src/Compilers/CSharp/Portable/Utilities/ValueSetFactory.NintValueSet.cs

```diff
@@ -53,7 +53,7 @@ public bool All(BinaryOperatorKind relation, int value)
                 return _values.All(relation, value);
             }
 
-            bool IValueSet.All(BinaryOperatorKind relation, ConstantValue value) => All(relation, value.Int32Value);
+            bool IValueSet.All(BinaryOperatorKind relation, ConstantValue value) => value.IsBad ? true : All(relation, value.Int32Value);
```

Comment:
Minor: `value.IsBad || All(...)`. Similar comment for other `nint` and `nuint` cases.